src/ui.js

```diff
@@ -6551,8 +6582,14 @@ document.addEventListener("DOMContentLoaded", () => {
     ui.initialise();
 
     const load_quiver_from_query_string = () => {
-        // If there is `q` parameter in the query string, try to decode it as a diagram.
         const query_data = query_parameters();
+
+        // If there is `e` in the query string, we want to treat the UI as running in embedded mode.
+        if (query_data.has("e")) {
```

Comment:
I would prefer to use a more descriptive query parameter like `embed`. (The `q` parameter is only short because it's expected to be included in LaTeX files, where it's helpful to be concise. In HTML documents, it's less of a problem.)